File: src/main/java/org/droidmate/analyzer/api/DummyApkMeta.java
package org.droidmate.analyzer.api;

import net.dongliu.apk.parser.bean.ApkMeta;

/**
 * Dummy APK metadata, used when no metadata is available on the APK
 */
public class DummyApkMeta extends ApkMeta {
    public DummyApkMeta() {
        super();

        this.setPackageName("DUMMY");
    }
}

File: src/main/kotlin/org/droidmate/analyzer/api/DummyApkMeta.kt
package org.droidmate.analyzer.api

import net.dongliu.apk.parser.bean.ApkMeta

/**
 * Dummy APK metadata, used when no metadata is available on the APK
 */
class DummyApkMeta : ApkMeta() {
    init {

        this.packageName = "DUMMY"
    }
}
